packages/interbit-ui-tools/src/middleware/sagas/index.js

```diff
@@ -29,17 +30,9 @@ function* loadInterbitSaga() {
     const { cli } = yield call(interbitContext)
 
     const config = yield call(getHtmlConfig, document)
-    yield put(actionCreators.initialConfig(config))
+    yield put.resolve(actionCreators.initialConfig(config))
 
-    const peers = getConfiguredPeers(config, { subtree: entireTree })
-    yield call(connectToPeers, { cli, peers })
-
-    const chainsToLoad = getConfiguredChains(config, {
```

Comment:
getConfiguredPeers / getConfiguredChains now read from state where needed, simplifying this